### PR TITLE
Gluon v2020.1

### PR DESCRIPTION
**Pull request by mweinelt**
Release notes written by @blocktrron and @mweinelt. So everyone else is invited to review.

@christf If you have significant babel changes that need to be mentioned please speak up.

### CODE REVIEW

File: docs/releases/v2020.1.rst [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+Gluon 2020.1 drops support for the IBSS wireless protocol. Communities
+still using IBSS are suggested to migrate to 802.11s using the scheduled
+domain switch.
+
+Performance enhancements
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Gluon now installs zram-swap by default on ``ar71xx`` devices with 8MB
+of flash and 32M of RAM.
```
Comment (CodeFetch) [inline docs/releases/v2020.1.rst]: 32MB as M = Megabits

File: docs/releases/v2020.1.rst [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+Target renaming
+~~~~~~~~~~~~~~~
+
+- The ``ipq40xx`` target was renamed to ``ipq40xx-generic``.
+- The ``ipq806x`` target was renamed to ``ipq806x-generic``.
+
+Status Page
+~~~~~~~~~~~
+
+The status page now includes information about the nexthop prettyname,
```
Comment (CodeFetch) [inline docs/releases/v2020.1.rst]: Now the status page or The status page includes ... now

File: docs/releases/v2020.1.rst [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+mac address and interface towards the selected gateway when batman-adv
```
Comment (CodeFetch) [inline docs/releases/v2020.1.rst]: MAC

File: docs/releases/v2020.1.rst [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+is used. It also shows the site and additionally the domain name if the
```
Comment (CodeFetch) [inline docs/releases/v2020.1.rst]: Additionally it displays

File: docs/releases/v2020.1.rst [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+node is in a multidomain setup.
+
+DECT button to enter config mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Many AVM devices don’t feature a separate RESET/WPS button, so starting
```
Comment (CodeFetch) [inline docs/releases/v2020.1.rst]: Thus starting with this release Gluon supports

File: docs/releases/v2020.1.rst [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+this release we support entering the config mode via DECT buttons.
+
+Bugfixes
+--------
+
+Autoupdater aliases
+~~~~~~~~~~~~~~~~~~~
+
+We have added several new aliases for continued autoupdater
```
Comment (CodeFetch) [inline docs/releases/v2020.1.rst]: continuing autoupdater compatibility for

File: docs/releases/v2020.1.rst [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+- Ubiquiti UniFi AC LR
+- Raspberry Pi
+
+Site changes
+------------
+
+site.mk
+~~~~~~~
+
+-  The ``GLUON_WLAN_MESH`` variable can be dropped, as 802.11s is now
```
Comment (CodeFetch) [inline docs/releases/v2020.1.rst]: is the only supported wireless transport protocol from now on.

File: docs/releases/v2020.1.rst [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+- ``make lint-sh`` to only check shell scripts
+- ``make lint-lua`` to only check lua scripts
+
+These require the shellcheck and luacheck tools. The docker image has
+been updated accordingly.
+
+Continuous integration
+~~~~~~~~~~~~~~~~~~~~~~
+
+We now utilize continuous integration testing using Jenkins and thereby
```
Comment (CodeFetch) [inline docs/releases/v2020.1.rst]: We utilize continuous integration testing using Jenkins now

(Either "time" at the beginning of the sentence if the sentence is long, but preferably "time" should be stated at the end of a sentence)

File: docs/releases/v2020.1.rst (hunk 5 above)
Comment (mweinelt) [reply docs/releases/v2020.1.rst]: "so" should be fine.

File: docs/releases/v2020.1.rst (hunk 3 above)
Comment (mweinelt) [reply docs/releases/v2020.1.rst]: Fixed.

File: docs/releases/v2020.1.rst (hunk 1 above)
Comment (mweinelt) [reply docs/releases/v2020.1.rst]: Fixed.

File: docs/releases/v2020.1.rst (hunk 5 above)
Comment (CodeFetch) [reply docs/releases/v2020.1.rst]: That's slang...

File: docs/releases/v2020.1.rst (hunk 4 above)
Comment (mweinelt) [reply docs/releases/v2020.1.rst]: "it" is already clear, so I just added "displays".

File: docs/releases/v2020.1.rst (hunk 6 above)
Comment (mweinelt) [reply docs/releases/v2020.1.rst]: dropping continued altogether.

File: docs/releases/v2020.1.rst (hunk 5 above)
Comment (CodeFetch) [reply docs/releases/v2020.1.rst]: "Therefore" would be fine.

File: docs/releases/v2020.1.rst (hunk 7 above)
Comment (mweinelt) [reply docs/releases/v2020.1.rst]: Fixed

File: docs/releases/v2020.1.rst (hunk 8 above)
Comment (mweinelt) [reply docs/releases/v2020.1.rst]: Do you have a reference for that rule?

File: docs/releases/v2020.1.rst (hunk 2 above)
Comment (mweinelt) [reply docs/releases/v2020.1.rst]: Same as below, I'd kindly ask for a reference on that.

File: docs/releases/v2020.1.rst (hunk 5 above)
Comment (mweinelt) [reply docs/releases/v2020.1.rst]: Ok

File: docs/releases/v2020.1.rst (hunk 8 above)
Comment (CodeFetch) [reply docs/releases/v2020.1.rst]: E.g. in your sentence you use "now" as an adverb, but it's usually an adjective. Your sentence is correct, but sounds strange.
https://wordtype.org/of/now

Comment (mweinelt) [reply docs/releases/v2020.1.rst]: I rebuilt the sentence.

File: docs/releases/v2020.1.rst (hunk 2 above)
Comment (mweinelt) [reply docs/releases/v2020.1.rst]: Reworded.

File: docs/releases/v2020.1.rst [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+introduces the ath79 target, which will replace ar71xx in the short
+term.
+
+Added Hardware support
+----------------------
+
+ar71xx-generic
+~~~~~~~~~~~~~~
+
+-  TP-Link Archer C60 v2
```
Comment (achterin) [inline docs/releases/v2020.1.rst]: I was under the impression that devices marked as broken would not be mentioned in the supported device list?

Comment (mweinelt) [reply docs/releases/v2020.1.rst]: Correct, good catch.

**Comment by NeoRaider** [comment]
`docs/conf.py` should be updated as well.

File: docs/releases/v2020.1.rst [hunk 10]
```diff
@@ -0,0 +1,189 @@
...
+-  TP-Link TL-WA801ND (v5)
+-  TP-Link TL-WR902AC (v3)
+
+Removed Hardware support
+------------------------
+
+-  ALFA Network Hornet-UB
+-  ALFA Network Tube2H
+-  ALFA Network N2
+-  ALFA Network N5
```
Comment (NeoRaider) [inline docs/releases/v2020.1.rst]: I think we discussed that we want to document the rationale for removed hardware support in the release notes.

File: docs/releases/v2020.1.rst [hunk 11]
```diff
@@ -0,0 +1,189 @@
+Gluon 2020.1
+============
+
+This is the first release of Gluon in 2020, based on OpenWrt 19.07. It
+introduces the ath79 target, which will replace ar71xx in the short
+term.
+
+Added Hardware support
```
Comment (NeoRaider) [inline docs/releases/v2020.1.rst]: Inconsistent capitalization (also below)

File: docs/releases/v2020.1.rst [hunk 12]
```diff
@@ -0,0 +1,189 @@
...
+still using IBSS are suggested to migrate to 802.11s using the scheduled
+domain switch.
+
+Performance enhancements
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+We install zram-swap by default on ``ar71xx`` devices with 8MB of flash
+and 32MB of RAM.
+
+Target renaming
```
Comment (NeoRaider) [inline docs/releases/v2020.1.rst]: I'd prefer the wording `Renamed targets`

File: docs/releases/v2020.1.rst [hunk 13]
```diff
@@ -0,0 +1,189 @@
...
+- Gateway nexthop information has been added to the statuspage when batman-adv
+  is used. This includes its MAC address and prettyname as well as the interface
+  name towards the selected gateway.
+- The site name has been added to the statuspage. If the node is in a multidomain
+  setup it will also show the domain name.
+
+DECT button to enter config mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Many AVM devices don’t feature a separate RESET/WPS button, therefore
```
Comment (NeoRaider) [inline docs/releases/v2020.1.rst]: Please use a regular single quote instead of `’`.

File: docs/releases/v2020.1.rst [hunk 14]
```diff
@@ -0,0 +1,189 @@
...
+   the only supported wireless transport from now on.
+
+Internals
+---------
+
+x86 partition size
+~~~~~~~~~~~~~~~~~~
+
+The x86 partition size has been reduced to fit on disk with 128 MB
+capacity.
```
Comment (NeoRaider) [inline docs/releases/v2020.1.rst]: This is a user-visible change, not an internal.

Comment (mweinelt) [reply docs/releases/v2020.1.rst]: Moved to major changes.

File: docs/releases/v2020.1.rst (hunk 10 above)
Comment (mweinelt) [reply docs/releases/v2020.1.rst]: Added

File: docs/releases/v2020.1.rst [hunk 15]
```diff
@@ -0,0 +1,191 @@
...
+~~~~~~~~~~~~~~~
+
+-  Aruba AP-303
+-  Aruba Instant On AP11
+-  AVM FRITZ!Repeater 1200
+
+ipq806x-generic
+~~~~~~~~~~~~~~~
+
+-  Netgcear R7800
```
Comment (herrbett) [inline docs/releases/v2020.1.rst]: There is a small typo here

Comment (mweinelt) [reply docs/releases/v2020.1.rst]: Thanks!

**Pull request merged**